addons/product/migrations/13.0.1.2/pre-migration.py

```diff
@@ -125,18 +125,32 @@ def insert_missing_product_template_attribute_value(env):
     )])
     openupgrade.logged_query(
         env.cr,
-        """
+        "ALTER TABLE product_template_attribute_value "
+        "ADD COLUMN attribute_line_id INT4",
+    )
+    fill_product_template_attribute_value_attribute_line_id(env)
```

Comment:
Please don't nest this high level methods.

Reply:
Ok, done.